### PR TITLE
Revise welcome message

### PR DESCRIPTION
**Pull request by RayWP**
As requested by @aldy505, i deleted some messages.
Also for the sake of diversity and Bhinneka Tunggal Ika, i add some languages to the welcome message

### CODE REVIEW

File: captcha/welcome.go [hunk 1]
```diff
@@ -65,29 +61,9 @@ Github. Pastiin profile Github-mu isinya project yang keren juga ya! Nggak usah
...
-"ini kenapa angkot depan rumah ngeselin banget sih", itu valid. Nggak ada topik OOT disini.
-
-Kamu nggak perlu pasang profile picture/username. Cukup matiin notification aja, berisik banget disini.
-
-Kita juga punya website dan Github organization. Bisa di cek di: https://teknologiumum.com
-dan https://github.com/teknologi-umum`,
-}
+	`您好 {user}! 
+	欢迎您在 Teknologi Umum, 我们每天都很嘈杂, 请把你的筒子声音关掉,
+	您要问什么, 请问吧. 希望您很高兴在这里`}
```
Comment (elianiva) [inline captcha/welcome.go:66]: mantap, jadi ada bahasa Stephanus biar makin diverse

**Comment by aldy505** [comment]
@RayWP bisa tolong tambahin lagi ngga messagenya? biar agak banyakan.

sama I noticed a bit of a new line problem on telegram. sekarang kan stringnya pake backtick, tolong ganti jadi normal double quotes aja. kalo ada newline tambahin `\n`.

contoh:
```go
var something = []string{
  "Hello this is" +
"not creating a new line but\n" +
"this will",
}
```

**Comment by RayWP** [comment]
Oke. Brti kayak Java style ya

**Comment by aldy505** [comment]
it's not Java style, it's C-style

**Pull request merged**